frontend/src/metabase/nav/containers/MainNavbar/MainNavbarView.tsx

```diff
@@ -85,18 +87,34 @@ function MainNavbarView({
   return (
     <SidebarContentRoot>
       <div>
+        <SidebarSection>
+          <ul>
+            <HomePageLink
+              isSelected={
+                selectedItem.type === "non-entity" && selectedItem.url === "/"
```

Comment:
We can now use `isNonEntityLinkSelected` instead of doing the check all over again